website/docs/r/continuous_compliance_notification.html.markdown

```diff
@@ -45,57 +45,89 @@ resource "dome9_continuous_compliance_notification" "test_notification" {
-    * `schedule_data` - Set or Get Notification schedule block supports:
-        * `cron_expression` - (Required) 
-        * `type` - (Required) Can be set to one of the following: "Detailed", "Summary", "FullCsv" or "FullCsvZip"
-        * `recipients` - (Required) 
+	if `sns_sending_state`  is Enabled, the following must be included:
+	
+	* `sns_data` - SNS notification details:
+        * `sns_topic_arn` - SNS topic ARN
+        * `sns_output_format` - SNS output format; can be  "JsonWithFullEntity", "JsonWithBasicEntity", or "PlainText".
     
-* `change_detection` - (Required) Change detection block supports:
-    * `email_sending_stat` - Can be one of the following: "Enabled" or "Disabled.
-    * `email_per_finding_sending_state` - Can be one of the following: "Enabled" or "Disabled.
-    * `sns_sending_state` - Can be one of the following: "Enabled" or "Disabled.
-    * `external_ticket_creating_state` - Can be one of the following: "Enabled" or "Disabled.
-    * `aws_security_hub_integration_state` - Can be one of the following: "Enabled" or "Disabled.
-    * `webhook_integration_state` - Can be one of the following: "Enabled" or "Disabled.
-    * `email_data` - Change detection block supports:
-        * `recipients` - (Required)
-    * `email_per_finding_data` - Email per finding data block supports:
-        * `recipients` - (Required)
-        * `notification_output_format` - (Required) Can be one of the following: "JsonWithFullEntity", "JsonWithBasicEntity", "PlainText".
-    * `sns_data` - SNS data block supports:
-        * `sns_topic_arn` - SNS topic arn
-        * `sns_output_format` - SNS output format, can be one of the following: "JsonWithFullEntity", "JsonWithBasicEntity", "PlainText".
-    * `ticketing_system_data` - Ticketing system data block supports:
-        * `system_type` - System type 
-        * `should_close_tickets` - Should close tickets 
-        * `domain` - Domain 
-        * `user` - User 
-        * `pass` - Pass 
-        * `project_key` - Project key 
-        * `issue_type` - Issue type 
-    * `aws_security_hub_integration` - AWS security hub integration block supports:
-        * `external_account_id` - (Required) external account id
-        * `region` - (Required) region
+	* `external_ticket_creating_state` - send each finding to an external ticketing system; can be  "Enabled" or "Disabled.
+    
+	if `external_ticket_creating_state`  is Enabled, the following must be included:
+	
+	* `ticketing_system_data` - Ticketing system details:
+        * `system_type` - system type; can be "ServiceOne", "Jira", or "PagerDuty"
+        * `should_close_tickets` - ticketing system should close tickets when resolved (bool)
+        * `domain` - ServiceNow domain name (ServiceNow only)
+        * `user` - user name (ServiceNow only)
+        * `pass` - password (ServiceNow only)
+        * `project_key` - project key (Jira) or API Key (PagerDuty)
+        * `issue_type` - issue type (Jira)
+	
+   
```

Comment:
Please remove unnecessary lines, one line is enough in order to separate between the fields